pkg/store/change/change.go

```diff
@@ -121,11 +85,11 @@ func CreateChange(config ValueCollections, desc string) (*Change, error) {
 	var pathList = make([]string, len(config))
 	// If a path is repeated then reject
 	for _, cv := range config {
-		for _, p := range pathList {
-			if cv.Path == p {
-				return nil, errors.New("Error Path " + p + " is repeated in change")
-			}
-		}
+		//for _, p := range pathList {
```

Comment:
why is this commented out ?

Reply:
good - catch - I had commented out when testing something and forgot to put it back in